pkg/openapi/go.mod

```
@@ -0,0 +1,3 @@
+module github.com/kubeflow/model-registry/pkg/openapi
+
+go 1.23.6
```

Is there a reason for this specific version? We had to have Go 1.23.6 for the kserve dependency, but I suspect this new module would be fine on a much older version of Go. Maybe just round it down to the major release:

```suggestion
go 1.23
```

---

@pboyd no specific reason. Just made it Go 1.23!